src/modules/core/transition_composite.c

```diff
@@ -144,6 +144,7 @@ void composite_line_yuv( uint8_t *dest, uint8_t *src, int width, uint8_t *alpha_
 			alpha_a += j;
 		if ( alpha_b )
 			alpha_b += j;
+		return;
```

Comment:
I think this will break for images with a width that is not a multiple of 8.

composite_line_yuv_sse2_simple can only operate on widths that are multiple of 8. The "for" loop below must run to process any remainder. For example, for a width of 10, composite_line_yuv_sse2_simple() will process 8 pixels and the for loop must run to process the last 2 pixels.

Can you test your proposed change with other widths that are not a multiple of 8?